include/apps/watchfaces/OswAppWatchfaceDigital.h

```diff
@@ -17,12 +17,12 @@ class OswAppWatchfaceDigital: public OswAppV2 {
     void onDraw() override;
     void onButton(Button id, bool up, ButtonStateNames state) override;
 
-    inline static uint8_t getDateFormat();  // Return 0 : mm/dd/yyyy 1 : dd.mm.yyyy 2 : yy.mm/dd
+    inline static uint8_t getDateFormat();
     static void refreshDateFormatCache();
     static void drawSteps();
     static void digitalWatch(short timeZone, uint8_t fontSize, uint8_t dateCoordY, uint8_t timeCoordY);
-    static void timeOutput(uint32_t hour, uint32_t minute, uint32_t second, bool showSecond = true);
-    static void dateOutput(uint32_t yearInt, uint32_t monthInt, uint32_t dayInt);
+    static void timeOutput(uint8_t hour, uint8_t minute, uint8_t second, bool showSecond = true);
+    static void dateOutput(uint16_t yearInt, uint8_t monthInt, uint8_t dayInt);
```

Comment:
type is `uint16_t` but param name is `nameInt`...lol

Reply:
OK